lib/tasks/dev.rake

```diff
@@ -135,6 +135,15 @@ require 'faker'
                 parent_id: reply.id
               )
           end
+          if rand <0.75
+            another_reply = new_reply.replies.create(
+              author_id: User.all.sample.id,  
```

Comment:
With the `belongs_to` association defined you can assign/query by record instead of id values if you want

changing this
```rb
author_id: User.all.sample.id,
```

to
```rb
author: User.all.sample,
```